pwndbg/commands/context.py

```diff
@@ -834,6 +834,56 @@ def context_args(with_banner=True, target=sys.stdout, width=None):
 last_signal: List[str] = []
 
 
+def get_thread_status(thread):
+    if thread.is_running():
+        return pwndbg.color.light_green("running")
+    elif thread.is_stopped():
+        return pwndbg.color.yellow("stopped")
+    elif thread.is_exited():
+        return pwndbg.color.gray("exited ")
+    else:
+        return "unknown"
+
+
+def context_threads(with_banner=True, target=sys.stdout, width=None):
+    threads = gdb.selected_inferior().threads()[::-1]
```

Comment:
We probably want to move some of this functionality to gdblib/threads.py - i.e. have a function there that will return data in a specific format/without GDB structures (this would allow us to have lldblib/threads.py returning a similar information in the future and supporting lldb this way)